src/locale/locales/de/messages.po

```diff
@@ -110,7 +102,7 @@ msgstr "Konto blockiert"
 
 #: src/view/com/profile/ProfileMenu.tsx:153
 msgid "Account followed"
-msgstr ""
+msgstr "Konto folgt"
```

Comment:
Not sure about the context here? Is it in the sense "you followed this account"?

Reply:
Yes this is the toast (pop-up notification) that is shown when you click/tap Follow on someone's profile. So "you followed this account" or alternatively "this account has been followed by you".